tests/app/main/test_briefs.py

```diff
@@ -676,11 +676,9 @@ def test_essential_requirements_evidence_escapes_brief_data(self):
         doc = html.fromstring(res.get_data(as_text=True))
         questions = doc.xpath(
             "//*[@class='question-heading']")
-        # if item is excaped correctly it will appear as text rather than an element
+
         assert questions[0].find("h1") is None
-        assert questions[0].text.strip() == '&lt;h1&gt;Essential one with xss&lt;/h1&gt;'
```

Comment:
![image](https://cloud.githubusercontent.com/assets/7228605/21645980/37dd658e-d28c-11e6-8c34-0f38e7e49ce3.png)
Have checked manually and they are not rendered as html elements but rendered as a string which means we have not been xss'd?